src/pages/City/City/City.tsx

```diff
@@ -271,6 +273,10 @@ const City = () => {
       },
     });
   }
+  async function SetAdmins(id: number){
+    const response = await getAllAdmins(id);
+    setAdminsAll(response.data.administration)
```

Comment:
Maybe it's worth wrap this response in try/catch/finally?

Reply:
getAllAdmins have try/catch